src/components/app/LandingPage.js

```diff
@@ -60,10 +60,10 @@ class LandingPage extends React.Component {
                 history.push("/" + key);
                 break;
             case "data-entry":
-                goToDhis2Url("/dhis-web-dataentry/index.action");
```

Comment:
I've refactored goToDhis2Url (see fc5adcd2), it's better to simply get the baseUrl from the d2 object. This also fixes those urls for a namespaces DHIS2 url (/2.30/...), which was not working.